LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-001.html

```diff
@@ -8,6 +8,7 @@
     <link rel="help" href="https://drafts.csswg.org/css-images-3/#color-stop-syntax">
     <link rel="help" href="https://drafts.csswg.org/css-images-3/#coloring-gradient-line">
     <meta name="assert" content="Tests that gradient with a single color stop renders the expected solid color">
+    <meta name="fuzzy" content="maxDifference=0-1; totalPixels=0-4000" />
```

Comment:
nit: here and in the rest of the files, can you match the other meta tags?
```suggestion
    <meta name="fuzzy" content="maxDifference=0-1; totalPixels=0-4000">
```